src/app/provider/view.nim

```diff
@@ -104,13 +104,37 @@ QtObject:
       }
     
     if SIGN_METHODS.contains(data.payload.rpcMethod):
-      return $ %* { # TODO: send transaction, return transaction hash, etc etc. Disabled in the meantime
-        "type": ResponseTypes.Web3SendAsyncCallback,
-        "messageId": data.messageId,
-        "error": {
-          "code": 4100
+      try:
+        let request = data.request.parseJson
+        let fromAddress = request["params"][0]["from"].getStr()
+        let to = request["params"][0]["to"].getStr()
+        let value = request["params"][0]["value"].getStr()
```

Comment:
I'm not 100% sure, but it is possible to send a transaction with no value, althrough maybe this case is captured by web3.js / ethers first, and will automatically set the value to 0?

Same thing with the nonce, in case the dapp that creates a transaction and sets an specific nonce (weird, but could happen :man_shrugging: )